lib/transaction/index.js

```diff
@@ -1025,32 +1040,33 @@ function acceptTraceContextPayload(traceparent, tracestate, transport) {
   }
 }
 
-/**
- * Parses incoming distributed trace header payload.
- *
- * @param {object} payload                - The distributed trace payload to accept.
- * @param {string} [transport='Unknown']  - The transport type that delivered the payload.
+/*
+  The following underscored functions are used exclusively by the
+  _acceptDistributedTracePayload method. They're broken out to reduce
+  its cognitive complexity.
  */
-Transaction.prototype._acceptDistributedTracePayload = _acceptDistributedTracePayload
-function _acceptDistributedTracePayload(payload, transport) {
+const _dtPayloadTest = function _dtPayloadTest(payload) {
   if (!payload) {
     this.agent.recordSupportability('DistributedTrace/AcceptPayload/Ignored/Null')
-    return
   }
-
+  return !!payload
+}
+const _isDtTest = function _isDtTest() {
   if (this.isDistributedTrace) {
     logger.warn(
       'Already accepted or created a distributed trace payload for transaction %s, ignoring call',
       this.id
     )
+    let supportabilityMetric = 'DistributedTrace/AcceptPayload/Ignored/CreateBeforeAccept'
     if (this.parentId) {
-      this.agent.recordSupportability('DistributedTrace/AcceptPayload/Ignored/Multiple')
-    } else {
-      this.agent.recordSupportability('DistributedTrace/AcceptPayload/Ignored/CreateBeforeAccept')
+      supportabilityMetric = 'DistributedTrace/AcceptPayload/Ignored/Multiple'
     }
-    return
+    this.agent.recordSupportability(supportabilityMetric)
+    return true
   }
-
+  return false
+}
+const _dtConfigTest = function _dtConfigTest() {
```

Comment:
I kind of wonder, is doing these function assignments merely to avoid having to give these functions docstrings?

Reply:
The function assignment is stylistic--I just prefer function assignments (no hoisting, for good or for ill). The declaration doesn't trigger the docstring requirement, likely because these aren't exported in any way for anything else to use them.

Reply:
Ah, I see, interesting.